internal/rbd/rbd_attach.go

```diff
@@ -65,12 +65,17 @@ const (
 	// `io-timeout` of rbd-nbd is to tweak NBD_ATTR_TIMEOUT. It specifies
 	// how long the IO should wait to get handled before bailing out.
 	setNbdIOTimeout = "io-timeout"
+
+	// rbd-nbd version with per device cookie support.
+	// TODO: Fix with right version when cookie changes merge.
+	rbdNbdCookieSupportVersion = "17.0.0"
```

Comment:
instead of depending on the version is it possible to check cookie is present in some help text of the CLI command?

Reply:
I was trying to address this in a more generic way, considering we might have to use the same utilities for other options in the future, today we are checking for `cookie` tomorrow we might want to check for other options or features.

Reply:
Won't this become problematic if someone backports the fix to another version? something like a downstream version?

Reply:
Upstream: I think this will be considered as a feature, and hence I don't think ceph will backport it to minor z-stream releases.

Downstream: In, downstream we should change this to something else ?

Reply:
it good to reuse the same code for both upstream and downstream instead of having downstream only change and I have seen few important bug fixes are backported from octopus to nautilus etc. and also if someone is maintaining their own ceph downstream it might break. just trying to cover all cases here instead of depending on the version?

Reply:
@nixpanic @Madhu-1 I understand your concern, and it kind of makes sense to me in a different angle, I was just thinking on the reusability part.

Okay! I will grep for `cookie` for now to satisfies your review :-)

BTW, do we want to keep the `IsVersionSatisfied` util func as I have also added `TestIsVersionSatisfied` may be for later use or just discard that work? (I'm just trying to save some future cycles)

Thanks!

Reply:
Thank you :+1:  if it's not used anywhere IMO it should be removed.